docs/conda-env.yml

```diff
@@ -0,0 +1,17 @@
+name: qtconsole_docs
+channels:
+- asmeurer
+dependencies:
+- python=3.4.3=2
+- nbformat
+- jupyter_client
+- ipykernel
+- numpydoc
+- qt
+- pexpect
+- pygments
+- pyzmq
+- pyqt
+- pip:
+    - recommonmark
```

Comment:
gotcha, thanks. Maybe a comment to that effect, so that later "Wait, we don't use that..." PRs don't come in and break it after we forget.